templates/vue/src/components/TapestryMap.vue

```diff
@@ -0,0 +1,324 @@
+            title: node.title,
+            accessible: node.accessible,
+            status: node.status,
+          })
+        }
+      }
+      return markers
+    },
+  },
+  watch: {
+    isSidebarOpen() {
+      // Let Leaflet know the container size has changed so it can adjust the map size
+      setTimeout(() => {
+        this.$refs.map.mapObject.invalidateSize()
+      }, 300)
+    },
+    selectedId(id) {
+      const marker = this.$refs["marker-" + id]
+      if (marker) {
+        marker[0].mapObject.fire("click")
+      }
+    },
+  },
+  methods: {
+    getCoord(coord, coordIfEmpty) {
+      return coord === "" ? coordIfEmpty : coord
+    },
+    updateZoom(zoom) {
+      this.currentZoom = zoom
+    },
+    updateCenter(center) {
+      this.currentCenter = center
+    },
+    hasMapCoordinates(node) {
+      if (node.mapCoordinates) {
+        return node.mapCoordinates.lat !== "" && node.mapCoordinates.lng !== ""
+      } else {
+        return false
+      }
+    },
+    selectNode(id) {
+      if (this.selectedId !== id) {
+        this.$router.push({
+          name: names.APP,
+          params: { nodeId: id },
+          query: this.$route.query,
+        })
+      }
+    },
+    openNode(id) {
+      this.$router.push({
+        name: names.LIGHTBOX,
+        params: { nodeId: id },
+        query: this.$route.query,
+      })
+    },
+    editNode(id) {
+      this.$router.push({
+        name: names.MODAL,
+        params: { nodeId: id, type: "edit", tab: "content" },
+        query: this.$route.query,
+      })
+    },
+    addNewNode() {
+      this.$router.push({
+        name: names.MODAL,
+        params: {
+          nodeId: this.empty ? 0 : this.rootId,
+          type: "add",
+          tab: "content",
+        },
+        query: this.$route.query,
+      })
+    },
+    editNodeCoordinates(id) {
+      this.$router.push({
+        name: names.MODAL,
+        params: { nodeId: id, type: "edit", tab: "coordinates" },
+        query: this.$route.query,
+      })
+    },
+    hasPermission(action) {
+      return Helpers.hasPermission(this.node, action)
+    },
+  },
+}
+</script>
+
+<style lang="scss" scoped>
+#map-container {
+  display: flex;
+
+  .nodes-list {
+    text-align: left;
+    width: 280px;
+
+    > .card {
+      margin-bottom: 10px;
+
+      &.selected {
+        background-color: #eee;
+      }
+
+      .card-body {
+        padding: 0.8em;
+      }
+    }
+
+    .add-new-node-btn {
+      background-color: #2c3e50;
+      background-color: #2c3e50;
+
+      &:hover {
+        border-color: #11a6d8;
+        background-color: #11a6d8;
+      }
+    }
+
+    + .map-content {
+      width: calc(100% -280px);
+    }
+  }
+
+  .map-content {
+    height: 900px;
+    width: 100%;
+    margin-bottom: -20%;
+
+    > div {
+      height: 80%;
+    }
+  }
+}
+</style>
+
+<style lang="scss">
+#map-container .nodes-list > .card .card-body table tbody > tr {
```

Comment:
This selector looks very specific - not super clear to me what element(s) this selector is actually affecting. If you're overriding bootstrap values, could a [scoped slot](https://bootstrap-vue.org/docs/components/table#custom-data-rendering) work better for this use case?

Reply:
There's only one table element in there. Not sure if the scoped slot can do what is intended as we are using the `stacked` table. This table doesn't provide much in terms of built-in modifiers, so it's necessary to make these minor modifications here to match the intended styling.